### ISSUE TITLE
Intel 7265 ac

### ISSUE THREAD

**Issue by lehmartins17** [AppleIntelWifi/device-logs#12]
[log.txt](https://github.com/AppleIntelWifi/device-logs/files/4396058/log.txt)
I want to congratulate you. His work is incredible and will solve the problems of many people with hackintosh. Follow my log.txt for your evaluation. I am extremely anxious for this kext to work.

**Comment by allansrc** [comment]
dude, did you get radio Sign?

**Comment by lehmartins17** [comment]
> dude, did you get radio Sign?

not yet. But waiting for update. Post your log.txt for developer help, please? :D

**Comment by allansrc** [comment]
Yep I'll do today. thanks.

**Comment by lehmartins17** [comment]
> Yep I'll do today. thanks.

You're welcome. I’m happy with your contribution, and I hope I even have Intel Wi-Fi up and running.

**Comment by lehmartins17** [comment]
No news for us at 7265. Unfortunately: '(

**Comment by i3p9** [comment]
@lehmartins17 hey I wanna submit a log for 7265 too. What's the procedure to submit logs, do I inject the kext first, then run the command to get the logs?

**Comment by allansrc** [comment]
> @lehmartins17 hey I wanna submit a log for 7265 too. What's the procedure to submit logs, do I inject the kext first, then run the command to get the logs?


`log show --last boot --predicate 'process == "kernel"' | grep -i intel > ~/Desktop/log.txt`

**Comment by lehmartins17** [comment]
> @lehmartins17 hey I wanna submit a log for 7265 too. What's the procedure to submit logs, do I inject the kext first, then run the command to get the logs?

yes, first -> kextload -v *.kext.
then open terminal and insert command: log show --last boot --predicate 'process == "kernel"' | grep -i intel > ~/Desktop/log.txt

**Comment by allansrc** [comment]
[log (1).txt](https://github.com/AppleIntelWifi/device-logs/files/4519627/log.1.txt)

**Comment by i3p9** [comment]
Alright all done. Here's the [log.txt](https://github.com/AppleIntelWifi/device-logs/files/4521596/log.txt)

**Comment by koluris** [comment]
Another one 
[log.txt](https://github.com/AppleIntelWifi/device-logs/files/4670623/log.txt)

**Comment by welllini** [comment]
and one more:
[log.txt](https://github.com/AppleIntelWifi/device-logs/files/4700676/log.txt)

**Comment by nemr0** [comment]
guys did it work?

**Comment by lehmartins17** [comment]
> guys did it work?

Yes, but work more or less... wifi down 30-35mbps wifi up 30-35mbps

**Comment by nemr0** [comment]
> > guys did it work?
> 
> Yes, but work more or less... wifi down 30-35mbps wifi up 30-35mbps

i can't build the kext can you send it

**Comment by lehmartins17** [comment]
Procure no YouTube por: AppleIntelWiFiV2.kext

**Comment by alex-spataru** [comment]
Hi, I also attach my [log.txt](https://github.com/AppleIntelWifi/device-logs/files/4810075/log.txt) file. 

My adapter is recognized successfully, however, no networks are found while scanning, and trying to manually input SSID and password does not work. I am currently running macOS Catalina on an HP Spectre (Clover/setup repo [here](https://github.com/alex-spataru/hackintosh-spectre-4101dx)).

My WiFi works using [itlwm](https://github.com/zxystd/itlwm) (recompiled with sdk for macOS Catalina). However, I would like to use this project since it works with the WiFi stack directly.

Greetings!

**Comment by alex-spataru** [comment]
@lehmartins17 did you do any changes to get the driver to work? I can load the kext successfully and can see the WiFi option in the settings app. However, no networks are shown while scanning (and manually inputting SSID/password fails too).

**Comment by princeofgiri** [comment]
I also wanna submit my log here [log.txt](https://github.com/AppleIntelWifi/device-logs/files/4902715/log.txt)

**Comment by alexandrmih89** [comment]
Hello, my log here (Big Sur 11.0.1) [log.txt](https://github.com/AppleIntelWifi/device-logs/files/5546705/log.txt)
